contexts/retailCoupon-context-v1.json

```diff
@@ -0,0 +1,108 @@
+{
+  "@context": {
+    "@protected": true,
+    "id": "@id",
+    "type": "@type",
+
+    "ClippedCouponCredential": {
+      "@id": "https://contexts.vcplayground.org/examples/retailCoupon-context-v1.json#ClippedCouponCredential"
+    },
+    "clippedCoupon": {
+      "@id": "https://contexts.vcplayground.org/examples/retailCoupon-context-v1.json#clippedCoupon",
+      "@type": "@id"
+    },
+    "name": {
+      "@id": "https://schema.org/name"
+    }
+  },
+
+  "CouponOffer": {
```

Comment:
There's a mistake in the context here where the closing bracket for `@context` is here and then `CouponOffer` is not nested inside the `@context` body.